hack-the-resume/src/components/Basics.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import '../styles/Basics.css'
+import logo from '../resume/assets/profile-pic.jpg'
+
+const Basics = (props) => {
+  return (
+    <div className='Basics'>
+      <img src={logo} className="Pic" alt="profile pic" />
+      <h2>{props.basics.name} | {props.basics.label}</h2>
+      <p>{props.basics.summary}</p>
+      <br/>
```

Comment:
My habit is to avoid using the br tag for styling purposes unless there is some semantic reason to be adding a line break